2024-08-21 14:57:26.096 +03:30 [INF] Request starting HTTP/2 GET https://localhost:7246/api/Walks?PageNumber=1&PageSize=1000 - null null
2024-08-21 14:57:26.269 +03:30 [INF] Executing endpoint 'NZWalks.API.Controllers.WalksController.GetAll (NZWalks.API)'
2024-08-21 14:57:26.310 +03:30 [INF] Route matched with {action = "GetAll", controller = "Walks"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] GetAll(System.String, System.String, System.String, System.Nullable`1[System.Boolean], Int32, Int32) on controller NZWalks.API.Controllers.WalksController (NZWalks.API).
2024-08-21 14:57:28.041 +03:30 [WRN] The query uses a row limiting operator ('Skip'/'Take') without an 'OrderBy' operator. This may lead to unpredictable results. If the 'Distinct' operator is used after 'OrderBy', then make sure to use the 'OrderBy' operator after 'Distinct' as the ordering would otherwise get erased.
2024-08-21 14:57:28.045 +03:30 [WRN] The query uses a row limiting operator ('Skip'/'Take') without an 'OrderBy' operator. This may lead to unpredictable results. If the 'Distinct' operator is used after 'OrderBy', then make sure to use the 'OrderBy' operator after 'Distinct' as the ordering would otherwise get erased.
2024-08-21 14:57:28.821 +03:30 [INF] Executed DbCommand (92ms) [Parameters=[@__p_0='?' (DbType = Int32), @__p_1='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT [t].[Id], [t].[DifficultyId], [t].[Discription], [t].[LengtjKm], [t].[Name], [t].[RegionId], [t].[WalkImageUrl], [d].[Id], [d].[Name], [r].[Id], [r].[Code], [r].[Name], [r].[RegionIMageUrl]
FROM (
    SELECT [w].[Id], [w].[DifficultyId], [w].[Discription], [w].[LengtjKm], [w].[Name], [w].[RegionId], [w].[WalkImageUrl]
    FROM [Walks] AS [w]
    ORDER BY (SELECT 1)
    OFFSET @__p_0 ROWS FETCH NEXT @__p_1 ROWS ONLY
) AS [t]
INNER JOIN [Difficulties] AS [d] ON [t].[DifficultyId] = [d].[Id]
INNER JOIN [Regions] AS [r] ON [t].[RegionId] = [r].[Id]
2024-08-21 14:57:29.065 +03:30 [INF] Executed action NZWalks.API.Controllers.WalksController.GetAll (NZWalks.API) in 2740.839ms
2024-08-21 14:57:29.072 +03:30 [INF] Executed endpoint 'NZWalks.API.Controllers.WalksController.GetAll (NZWalks.API)'
2024-08-21 14:57:29.156 +03:30 [ERR] 8f8a0d20-99de-42e6-a927-1965d5822b2a : This is a new Exception
System.Exception: This is a new Exception
   at NZWalks.API.Controllers.WalksController.GetAll(String Filteron, String FilterQuery, String SortBy, Nullable`1 IsAscending, Int32 PageNumber, Int32 PageSize) in C:\Users\User\Desktop\c# project\NZWallks\NZWallks.API\Controllers\WalksController.cs:line 43
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at NZWalks.API.MiddleWares.ExceptionHandlerMiddleware.InvokeAsync(HttpContext httpContext) in C:\Users\User\Desktop\c# project\NZWallks\NZWallks.API\MiddleWares\ExceptionHandlerMiddleware.cs:line 23
2024-08-21 14:57:29.298 +03:30 [INF] Request finished HTTP/2 GET https://localhost:7246/api/Walks?PageNumber=1&PageSize=1000 - 500 null application/json; charset=utf-8 3205.7143ms
